data/config_files/function_list_central_qc_run.yml

```diff
@@ -27,6 +27,7 @@
 - qc_genotype
 - qc_verify_bam_id
 - qc_upstream_tags
+- qc_rna_seqc
```

Comment:
Don't think we should add it in the "qc" runs. These are the paired 25bp runs used for quantification of libraries (I'll submit a separate PR to remove other inappropriate QC checks).